### PR TITLE
VasiliZas

### PR DESCRIPTION
**Pull request by VasiliZas**

### CODE REVIEW

**Review by trump-ny — DISMISSED** [review]

File: homework/Vasili_Zasinets/hw0709/decoder/Encoder.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.tutrit.java.quickstart.hw0709.decoder;
+
+public class Encoder {
+
+
+    void decode(char symbol) {
+        int code = (int) symbol;
+        System.out.println("code = " + code);
```
Comment (lamaconda) [inline homework/Vasili_Zasinets/hw0709/decoder/Encoder.java:8]: better use logger

**Review by lamaconda — CHANGES_REQUESTED** [review]

File: homework/Vasili_Zasinets/hw0709/decoder/EncoderDemo.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.tutrit.java.quickstart.hw0709.decoder;
+
+public class EncoderDemo {
+
+    public static void main(String[] args) {
+        var encoder = new Encoder();
+        encoder.decode('X');
+        encoder.encode((short) 65);
+    }
```
Comment (max-java) [inline homework/Vasili_Zasinets/hw0709/decoder/EncoderDemo.java:9]: This should be moved to tests

File: homework/Vasili_Zasinets/hw0713/Butterfly.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.tutrit.java.quickstart;
+
+public class Butterfly {
+    public static void main(String[] args) {
+        int[][] butterfly = {{1, 1, 1, 1, 1, 1, 1}, {0, 1, 1, 1, 1, 1, 0}, {0, 0, 1, 1, 1, 0, 0},
+                {0, 0, 0, 1, 0, 0, 0}, {0, 0, 1, 1, 1, 0, 0}, {0, 1, 1, 1, 1, 1, 0}, {1, 1, 1, 1, 1, 1, 1}};
+        int i;
+        int k;
+        for (k = 0; k < butterfly.length; k++) {
+            for (i = 0; i < butterfly.length; i++) {
```
Comment (max-java) [inline homework/Vasili_Zasinets/hw0713/Butterfly.java:10]:
```
for (int k = 0; k < butterfly.length; k++) {
            for (int i = 0; i < butterfly.length; i++) {
```

File: homework/Vasili_Zasinets/hw0713/Butterfly.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+                System.out.print(butterfly[k][i]);
```
Comment (max-java) [inline homework/Vasili_Zasinets/hw0713/Butterfly.java:11]: logger should be used

File: homework/Vasili_Zasinets/hw0713/Butterfly.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
+package com.tutrit.java.quickstart;
+
+public class Butterfly {
+    public static void main(String[] args) {
+        int[][] butterfly = {{1, 1, 1, 1, 1, 1, 1}, {0, 1, 1, 1, 1, 1, 0}, {0, 0, 1, 1, 1, 0, 0},
+                {0, 0, 0, 1, 0, 0, 0}, {0, 0, 1, 1, 1, 0, 0}, {0, 1, 1, 1, 1, 1, 0}, {1, 1, 1, 1, 1, 1, 1}};
+        int i;
```
Comment (max-java) [inline homework/Vasili_Zasinets/hw0713/Butterfly.java:7]: looks like test case, should be moved to tests

**Review by max-java — CHANGES_REQUESTED** [review]
please, use quickstart template and run sonarQube analyse before pull request

**Pull request closed without merging**